Formula/fly.rb

```diff
@@ -0,0 +1,23 @@
+class Fly < Formula
+  desc "Command line interface to Concourse"
+  homepage "https://concourse.ci"
+  url "https://github.com/concourse/concourse.git",
+    :tag => "v2.3.1",
+    :revision => "0fc7e2f6d46412ea70c8a9d81671c66d27276e44"
+
+  bottle :unneeded
+  
+  depends_on "go"
+
+  def install
+    cd "src/github.com/concourse/fly" do
```

Comment:
`cd "src/github.com/concourse/fly"` and then skip the `do`, `end` and indentation.